lib/es/esClient.js

```diff
@@ -117,6 +117,24 @@ async function deleteIndex(index) {
   }
 }
 
+async function removeOrphanedIndexes(alias) {
+  const allIndexes = await client.indices.getAlias();
+  const orphans = Object.keys(allIndexes).filter(key => key.startsWith(`${alias}_`) && !allIndexes[key].aliases[alias]);
+  /* eslint-disable */
+  const errors = [];
+  for (const orphan of orphans) {
```

Comment:
Rather than looping through the orphans could this be replaced with a call to `Promises.all()`? The reason for doing so would be to use the built-in construct for managing multiple promise execution rather than having to do it manually.

Reply:
If there were a thousand orphaned indexes I'd be concerned about the effect of running 1000 hits on ES at around the same time.

We could get the best of both worlds by batching the deletes into promise.all blocks of 10, but for the (hopefully) small number of orphans we have the current implementation should be performant enough and not cause undue load on the cluster.